src/cmd/Makefile.am

```diff
@@ -16,6 +16,7 @@ common_ldadd = \
 	$(top_builddir)/src/libnpclient/libnpclient.a \
 	$(top_builddir)/src/libnpfs/libnpfs.a \
 	$(top_builddir)/src/liblsd/liblsd.a \
+	$(top_builddir)/src/libdiod/libdiod.a \
```

Comment:
commit message typo: "linkagae"

Reply:
Thanks! Pushed that fix and I'll set MWP.